tasks/cluster_property.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Ensure pcs_property are present.
+  pcs_property:
+    name: "{{ item.name }}"
+    state: "{{ item.state | default(omit) }}"
+    node: "{{ item.node | default(omit) }}"
+    value: "{{ item.value | default(omit) }}"
+    cib_file: "{{ resource.cib_file | default(omit) }}"
```

Comment:
(medium) - `resource.cib_file` variable is not defined, most probably this is `item.cib_file`.

Reply:
Cib file have been removed on the commit accea57 as you suggest

Reply:
looks good.